Extension/src/common.ts

```diff
@@ -603,6 +607,7 @@ export async function checkInstallJsonsExist(): Promise<boolean> {
         if (!await checkFileExists(path.join(extensionPath, file))) {
             installJsonsExist = false;
             console.log(`Extension file ${file} is missing.`);
+            Telemetry.logLanguageServerEvent("missingJson", { "source": `${file}` });
```

Comment:
Same here. Just log the name of the file instead of the full path.

Reply:
the file name used here already a base name (not a full path)